.circleci/create_circleci_config.py

```diff
@@ -156,6 +207,7 @@ def job_name(self):
         "pip install .[sklearn,torch,testing,sentencepiece,torch-speech,vision,timm]",
         "pip install git+https://github.com/huggingface/accelerate",
     ],
+    parallelism=1,
```

Comment:
I set it back to `1` (well I can remove this line too, as `None` is the same as `1`), as @sgugger is not very happy with the test reports being splitted. We can change this value in the future once we really need N > 1 or after I find a good way to combine the test reports in this case.